excel-importer/src/com/axonivy/util/excel/importer/ExcelReader.java

```diff
@@ -87,11 +89,41 @@ private static void updateColumn(Column column, Cell cell) {
         && !CellUtils.isInteger(cell)) {
       column.setType(Double.class);
     }
+    if (cell.getCellType() == CellType.STRING) {
+      column.setType(String.class);
+    }
     if (column.getType().equals(String.class)) {
-      var cellLength = cell.getStringCellValue().length();
-      if (cellLength > column.getDatabaseFieldLength()) {
-        column.setDatabaseFieldLength(cellLength);
+      var cellValue = getCellValueAsString(cell);
+      if (cellValue.length() > (column.getDatabaseFieldLength() == null ? 0 : column.getDatabaseFieldLength())) {
```

Comment:
just return the default condition since case column db length = null is somehow covered by above feedback